5-app-infra/common.auto.example.tfvars

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+org_id = "000000000000"
+
+terraform_service_account = "org-terraform@example-project-2334.iam.gserviceaccount.com"
```

Comment:
The `terraform_service_account` here should be the SA created in 4-projects. For instance https://github.com/terraform-google-modules/terraform-example-foundation/blob/0d315491bc3ad7b9578b40042ec97907c4b69cd5/4-projects/business_unit_1/development/outputs.tf#L22 for deploying to base_shared